nodejs/src/wrapper.ts

```diff
@@ -46,11 +46,60 @@ const { NetInstrumentation } = require('@opentelemetry/instrumentation-net');
 const { PgInstrumentation } = require('@opentelemetry/instrumentation-pg');
 const { RedisInstrumentation } = require('@opentelemetry/instrumentation-redis');
 
+// AWS lambda instrumentation response hook for Server-Timing support
+function getEnvBoolean(key: string, defaultValue = true) {
+  const value = process.env[key];
+
+  if (value === undefined) {
+    return defaultValue;
+  }
+
+  if (['false', 'no', '0'].indexOf(value.trim().toLowerCase()) >= 0) {
+    return false;
+  }
+
+  return true;
+}
+
+// an educated guess - how to check if particular response is an API Gateway event?
+function isApiGatewayResponse(data:any) {
+    return (data.res && data.res.body && data.res.statusCode);
```

Comment:
```suggestion
    return (data && data.res && data.res.body && data.res.statusCode);
```

Reply:
Data will never be null here with current instrumentation. But it's good to be extra safe :)